src/log/log.cr

```diff
@@ -33,33 +33,41 @@ class Log
     @backend = value
   end
 
-  {% for method, severity in {
-                               trace:  Severity::Trace,
-                               debug:  Severity::Debug,
-                               info:   Severity::Info,
-                               notice: Severity::Notice,
-                               warn:   Severity::Warn,
-                               error:  Severity::Error,
-                               fatal:  Severity::Fatal,
-                             } %}
-    # Logs a message if the logger's current severity is lower or equal to `{{severity}}`.
+  {% for method in [ "trace", "debug", "info", "notice", "warn", "error", "fatal" ] %}
```

Comment:
Optional suggestion for more concise code:
```suggestion
  {% for method in %w(trace debug info notice warn error fatal) %}
```

Reply:
oh i like that